From 1b46eb1cc02fa6e97c409515b498a750c5028695 Mon Sep 17 00:00:00 2001
From: Luca Stefani <luca.stefani.ge1@gmail.com>
Date: Tue, 5 Feb 2019 09:41:03 +0100
Subject: [PATCH] Squash of
 github/lineage-16.0(a8c2da7)..lineage-16.0-android-9.0.0_r31(d11eac8) (DO NOT
 SUBMIT)

Brad Ebinger (1):
 - Update carrier configs to use new call barring keys

Luca Stefani (1):
 - Merge tag 'android-9.0.0_r31' into lineage-16.0-android-9.0.0_r31

android-build-team Robot (1):
 - Snap for 5044688 from 173eb6637fd568b2b9c3ab787782a9418407c5dc to
   pi-qpr2-release

Change-Id: I9b632482ca0a493e86bafe38bfb44b9f08eca211

Change-Id: I72a6ca075213751edd4871973bccaaef74509c4f
---
 assets/carrier_config_44020.xml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/assets/carrier_config_44020.xml b/assets/carrier_config_44020.xml
index a84d4c9..7f7dc55 100644
--- a/assets/carrier_config_44020.xml
+++ b/assets/carrier_config_44020.xml
@@ -15,5 +15,7 @@
         <string name="default_vm_number_string">1416</string>
         <long name="message_expiration_time_long" value="3600000" />
         <boolean name="ignore_data_enabled_changed_for_video_calls" value="true" />
+        <boolean name="call_barring_supports_password_change_bool" value="false" />
+        <boolean name="call_barring_supports_deactivate_all_bool" value="false" />
     </carrier_config>
 </carrier_config_list>
-- 
2.17.1

